specification/cosmos-db/resource-manager/readme.md

```diff
@@ -45,11 +59,25 @@ modelerfour:
   lenient-model-deduplication: true
 ```
 
+
+### Tag: package-2021-06
+
```

Comment:
This is duplicate.

Reply:
I created https://github.com/Azure/azure-rest-api-specs/issues/14995 for this duplicate tag.